plugins/modules/ipaautomember.py

```diff
@@ -72,9 +72,9 @@
     elements: dict
     aliases: ["automemberinclusiveregex"]
   action:
-    description: Work on service or member level
-    default: service
-    choices: ["member", "service"]
+    description: Work on automember or member level
+    default: automember
+    choices: ["member", "automember"]
```

Comment:
I think automemberrule is preferable instead of automember.

Reply:
The default action is always the module name without the "ipa" prefix.

I agree that the name automemberrule would be a lot better, but the IPA API is also using "automember" without "rule".

Reply:
Sure, No issues